src/codeCache.cpp

```diff
@@ -168,6 +168,20 @@ const void* CodeCache::findSymbolByPrefix(const char* prefix, int prefix_len) {
     return NULL;
 }
 
+const void* CodeCache::findSymbolByPrefix(const char* prefix, int prefix_len, char second_choice_if_contains) {
+    int second_choice = -1;
+    for (int i = 0; i < _count; i++) {
+        const char* blob_name = _blobs[i]._name;
+        if (blob_name != NULL && strncmp(blob_name, prefix, prefix_len) == 0) {
+            if (strchr(blob_name, second_choice_if_contains) == NULL) {
```

Comment:
`blob_name + prefix_len` to avoid scanning the prefix itself?

Reply:
70494a561c8882895115806f117746ba9b35318f